COMPILING.md

```diff
@@ -463,7 +463,7 @@ Visual Studio 2015 is required to build Cataclysm. We created solution and proje
 
 We've prepared an archive containing all the headers and libraries required to build Cataclysm: [http://dev.narc.ro/cataclysm/WinDepend-MSVC.zip](http://dev.narc.ro/cataclysm/WinDepend-MSVC.zip) or [http://dev.narc.ro/cataclysm/WinDepend-MSVC.7z](http://dev.narc.ro/cataclysm/WinDepend-MSVC.7z). The latter is smaller, but if you don't have a 7-zip archive extracter, the former one is easier to deal with.
 
-Extract the 'WinDepend' folder and put it in the root folder of Cataclysm project.
+Extract the 'WinDepend' folder and put it in the root folder of Cataclysm project, run the enclosed `copy_dll_to_bin.bat` to put all the dlls in Windepend\bin\{x86,x64}.
```

Comment:
The new file is called `copydll.bat`, what does this `copy_dll_to_bin.bat` refer to?